pkg/runs/runsGet.go

```diff
@@ -46,6 +46,7 @@ func GetRuns(
 	runName string,
 	age string,
 	requestor string,
+	result string,
```

Comment:
It's odd passing a variable called result into a function ?
Perhaps resultFilter would be clearer ?

requestor might be better as requestorFilter also ?